operator/apis/platform/v1alpha1/central_types.go

```diff
@@ -125,7 +125,7 @@ type CentralComponentSpec struct {
 	Telemetry *Telemetry `json:"telemetry,omitempty"`
 
 	// Configures resources within Central in a declarative manner.
-	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=8,displayName="Declarative Configuration",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:hidden"}
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=8,displayName="Declarative Configuration"
```

Comment:
Is it necessary to expose this config in the UI?
If not, I suggest not adding it to the UI. It is only used for demo use-cases, maintaining the UI is too expensive for it not being used for complex configuration scenarios in production systems.

Reply:
Hmm, I found it quite neat to be able to select config maps / secrets directly within the UI of the operator.
I found it to be at the very least somewhat more prominently discoverable, but if it's too much maintenance to have it, I'm OK with removing this completely.

So, removing it then I suppose?

Reply:
> So, removing it then I suppose?

Yes, this field must not being displayed in the UI and can stay hidden.